src/components/atoms/StoryCardFullWidth/index.js

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { get } from "lodash";
+
+import { Link } from "@quintype/components";
+import Contributor from "../../atoms/Contributor";
+import ResponsiveImageWithFallback from "../ResponsiveImageWithFallback";
+import { getStoryData, generateImageSources } from "../../utils/utils";
+import Icon from "../Icon";
+import { getMonthDayString } from "../../utils/time-format";
+// import { isPremium } from "../../../../isomorphic/data/story";
+
+import "./styles.m.css";
+
+export default function StoryCardFullWidth({ story, className = "", cardWithImageZoom = true }) {
+  const storyData = getStoryData(story);
+
+  if (!(storyData.headline && story.url)) {
+    return null;
+  }
+
+  const contributor = get(story, ["authors", 0]);
+  const contributorRole = get(story, ["authors", 0, "contributor-role", "name"], "");
+  const externalLink = get(story, ["metadata", "reference-url"]);
+
+  const icons = {
+    "photo-album": "gallery",
+    photo: "photostory",
+    video: "video"
+  };
+
+  const iconType = get(icons, [story["story-template"]], null);
+  return (
+    <div styleName="base" className={`${className} ${cardWithImageZoom ? "card-with-image-zoom" : ""}`}>
+      <Link
+        aria-label={`${"Read full story: "} ${storyData.headline}`}
+        styleName="link"
+        href={externalLink || story.url}
+        externalLink={externalLink}
+      >
+        <ResponsiveImageWithFallback
+          styleName="image-container"
+          slug={storyData.imageS3Key}
+          metadata={storyData.imageMetadata}
+          alt={storyData.imageCaption}
+          imgParams={{ auto: ["format", "compress"] }}
+          sources={generateImageSources(
+            { aspectRatio: [16, 9], screenWidthCoverage: 1 },
+            { aspectRatio: [16, 9], screenWidthCoverage: 0.66 }
+          )}
+        >
+          {iconType && (
+            <span styleName="story-template-icon">
+              <Icon type={iconType} styleName="template-icon" />
+            </span>
+          )}
+          {story.engagement && story.engagement.total && (
+            <span styleName="engagement-total">
+              <Icon type={"speaker"} styleName="speaker-icon" />
+              {story.engagement.total}
+            </span>
+          )}
+        </ResponsiveImageWithFallback>
+        <div styleName="content">
+          {contributor && (
+            <Contributor
+              name={contributor["name"]}
+              type={contributorRole}
+              iconColor="#929292"
+              styleName="contributor"
```

Comment:
this i think should be the className

Reply:
I think this should be styleName. In storyCardFullWidth component I am adding margin property that is local for that component.

Reply:
can be discussed when you are in

Reply:
let me think of it